config.toml.sample

```diff
@@ -22,6 +22,7 @@ maskUserInfo = false                    # If true, then account information in h
 enableContainerCommit = false           # If true, then user can request container commit corres ponding to the session in running status
 hideAgents = true                       # If false, show the `Agent Summary` menu in the sidebar.
 enable2FA = false                       # If true and backend.ai-totp-plugin is installed, users can activate 2FA.
+force2FA = false                        # If true, user should be register the 2FA to use Backend.AI WebUI.
```

Comment:
We need to add this to webserver configuration, too.
Every new key added to the configuration has to be reflected [here](https://github.com/lablup/backend.ai/blob/main/configs/webserver/sample.conf), accordingly.

Reply:
@Sujin-Kim1  please create a related PR for this in [Backend.AI Mono repo](https://github.com/lablup/backend.ai).

Reply:
@lizable I already wrote it to the PR description. https://github.com/lablup/backend.ai/pull/1161